web-console/4.0-designs/status/status.md

```diff
@@ -47,3 +47,13 @@ Note: Icons should never be animated in a list view.
 * [Color](https://www.patternfly.org/styles/color-palette/)
   * **Black**: pf-black #030303
   * **Blue**: pf-blue-400 #0088CE (for links only)
+
+## Status Popovers
+
+If additional information about a status is available, a popover should be used to provide context and up to two suggested actions. Statuses with popovers should be styled in the same way as inline links to indicate interactivity.
```

Comment:
@andybraren why are we limiting to 2?  might it make sense to allow a primary as a link, but then offer a kebab or some other pattern to provide an overflow mechanism?

Reply:
I softened the wording of this to allow "one or more suggested actions" in order to accommodate future use cases.

I haven't personally encountered a situation where more than 2 actions (not links, which could be included in the body) were relevant to a particular status, but I suppose that could happen.